index.js

```diff
@@ -148,7 +148,7 @@ class SortableListView extends React.Component {
     const currentPanValue = { x: 0, y: 0 }
 
     this.state = {
-      ds: new ListView.DataSource({
+      ds: new (props.ListViewComponent || ListView).DataSource({
```

Comment:
I think it should be `(props.ListViewComponent && props.ListViewComponent.DataSource) || ListView.DataSource`?

Reply:
Feel free to send a pull request. It is expected in this case that when you pass in a custom list view component, that it will have a DataSource field.